examples/notebooks/Desbordante_demo_scenario_3_anomaly_detection.ipynb

```diff
@@ -0,0 +1,437 @@
+            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m4.0/4.0 MB\u001b[0m \u001b[31m27.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
+            "\u001b[?25hInstalling collected packages: desbordante\n",
+            "Successfully installed desbordante-2.3.2\n",
+            "--2025-03-20 18:46:41--  https://raw.githubusercontent.com/Desbordante/desbordante-core/refs/heads/main/examples/datasets/cargo_data_1.csv\n",
+            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n",
+            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 4139 (4.0K) [text/plain]\n",
+            "Saving to: ‘cargo_data_1.csv’\n",
+            "\n",
+            "cargo_data_1.csv    100%[===================>]   4.04K  --.-KB/s    in 0s      \n",
+            "\n",
+            "2025-03-20 18:46:41 (32.1 MB/s) - ‘cargo_data_1.csv’ saved [4139/4139]\n",
+            "\n",
+            "--2025-03-20 18:46:41--  https://raw.githubusercontent.com/Desbordante/desbordante-core/refs/heads/main/examples/datasets/cargo_data_2.csv\n",
+            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n",
+            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 4249 (4.1K) [text/plain]\n",
+            "Saving to: ‘cargo_data_2.csv’\n",
+            "\n",
+            "cargo_data_2.csv    100%[===================>]   4.15K  --.-KB/s    in 0.001s  \n",
+            "\n",
+            "2025-03-20 18:46:41 (3.19 MB/s) - ‘cargo_data_2.csv’ saved [4249/4249]\n",
+            "\n",
+            "--2025-03-20 18:46:41--  https://raw.githubusercontent.com/Desbordante/desbordante-core/refs/heads/main/examples/datasets/cargo_data_3.csv\n",
+            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...\n",
+            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 4249 (4.1K) [text/plain]\n",
+            "Saving to: ‘cargo_data_3.csv’\n",
+            "\n",
+            "cargo_data_3.csv    100%[===================>]   4.15K  --.-KB/s    in 0s      \n",
+            "\n",
+            "2025-03-20 18:46:41 (60.0 MB/s) - ‘cargo_data_3.csv’ saved [4249/4249]\n",
+            "\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Setting up various algorithm parameters."
```

Comment:
I believe somewhere at the start we need to put text cell which will say:

This collection of scenarios demonstrates how to solve various data quality problems by exploiting patterns found (or validated) by Desbordante.

In this scenario, we showcase a simple application that performs anomaly detection in a table.

The idea of this scenario is described in the paper "Solving Data Quality Problems with Desbordante: a Demo" by G. Chernishev et al., available at https://arxiv.org/abs/2307.14935. There is also an interactive demo at https://desbordante.streamlit.app/.